core/sockets.py

```diff
@@ -72,6 +92,34 @@ class SvSocketCommon:
 
     quicklink_func_name: StringProperty(default="", name="quicklink_func_name")
 
+    allow_flattern : BoolProperty(default = False)
+    allow_simplify : BoolProperty(default = False)
+
+    use_graft : BoolProperty(
```

Comment:
Would you like to move all this new fields and methods in separate mixin class?

Reply:
I think all Sverchok socket classes should have such possibilities, no?

Reply:
I meant something like this

```Python
class NewClass: ...

class SvSocketCommon(NewClass): ...
```
It will let separate new features and in future it will be easier to handle them.